TermExcelerator/Sdl.Community.TermExcelerator/Allow.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sdl.Community.TermExcelerator
+{
+	public class Allow
+	{
+		private static bool Access;
+		public Allow(bool access) 
+		{ 
+			Access = access;
+		}
+
+		public Allow() { }
+
+		public bool Allowaccess { get { return Access; } set {  Access = value; } }
```

Comment:
I don't see it used (Allowaccess). Is this used externally?
I would also recommend calling it AllowAccess - unless this name (Allowaccess) is already set in stone